pkg/agent/apiserver/handlers/bgproute/handler.go

```diff
@@ -72,11 +73,13 @@ func HandleFunc(bq querier.AgentBGPPolicyInfoQuerier) http.HandlerFunc {
 
 		var bgpRoutesResp []apis.BGPRouteResponse
 		for bgpRoute := range bgpRoutes {
-			bgpRoutesResp = append(bgpRoutesResp, apis.BGPRouteResponse{
-				Route:     bgpRoute.Prefix,
-				Type:      string(bgpRoutes[bgpRoute].Type),
-				K8sObjRef: bgpRoutes[bgpRoute].K8sObjRef,
-			})
+			if bgpRouteType == "" || bgpRouteType == string(bgpRoutes[bgpRoute].Type) {
```

Comment:
why not use:
```go
for route, metadata := range bgpRoutes
```
or
```go
for route := range bgpRoutes {
        metadata := &bgpRoutes[route]
```
instead of having `bgpRoutes[bgpRoute]` in multiple places?